src/components/marketing/icon__build-deploy.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
No need to include SVGs in this repository we have https://github.com/RedHatInsights/frontend-assets repository to host these files.